fedimint-core/src/module/mod.rs

```diff
@@ -110,10 +110,15 @@ impl ApiRequestErased {
     }
 }
 
-/// Authentication uses the hashed user password in PHC format
+/// Authentication should be the hashed user password in PHC format
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct ApiAuth(pub String);
 
+/// Decryption key should be the hashed user password (different salt than the
+/// `ApiAuth`) to encrypt the private key material in configs
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct DecryptionKey(pub [u8; 32]);
```

Comment:
This doesn't say anything about what for is this decryption key, or type. Is it supposed to be a generic newtype over any decryption key we might use?

Reply:
The decryption key can be derived client-side using any technique, but it's purpose will be "to encrypt the private key material in configs" as in the comment.  Maybe I should rename it `ConfigDecryptionKey`?

Reply:
> Maybe I should rename it ConfigDecryptionKey?

+ a comment and I think that's good.